lib/matplotlib/figure.py

```diff
@@ -1261,127 +1261,216 @@ def draw_artist(self, a):
             settings.
 
-          markerfirst: [ *True* | *False* ]
-            if *True*, legend marker is placed to the left of the legend label
-            if *False*, legend marker is placed to the right of the legend
-            label
+        markerfirst : bool
+            If *True*, legend marker is placed to the left of the legend label.
+            If *False*, legend marker is placed to the right of the legend
+            label.
 
-          frameon: [ *None* | bool ]
+        frameon : None or bool
             Control whether the legend should be drawn on a patch (frame).
             Default is *None* which will take the value from the
             ``legend.frameon`` :data:`rcParam<matplotlib.rcParams>`.
 
-          fancybox: [ *None* | *False* | *True* ]
-            if *True*, draw a frame with a round fancybox.  If *None*, use rc
+        fancybox : None or bool
+            If *True*, draw a frame with a round fancybox. If *None*, use rc
+            settings.
 
-          shadow: [ *None* | *False* | *True* ]
+        shadow : None or bool
             If *True*, draw a shadow behind legend. If *None*, use rc settings.
 
-          framealpha: [ *None* | float ]
+        framealpha : None or float
             Control the alpha transparency of the legend's background.
             Default is *None* which will take the value from the
             ``legend.framealpha`` :data:`rcParam<matplotlib.rcParams>`.
 
-          facecolor: [ *None* | "inherit" | a color spec ]
+        facecolor : None or "inherit" or a color spec
             Control the legend's background color.
             Default is *None* which will take the value from the
             ``legend.facecolor`` :data:`rcParam<matplotlib.rcParams>`.
             If ``"inherit"``, it will take the ``axes.facecolor``
             :data:`rcParam<matplotlib.rcParams>`.
 
-          edgecolor: [ *None* | "inherit" | a color spec ]
+        edgecolor : None or "inherit" or a color spec
             Control the legend's background patch edge color.
             Default is *None* which will take the value from the
             ``legend.edgecolor`` :data:`rcParam<matplotlib.rcParams>`.
             If ``"inherit"``, it will take the ``axes.edgecolor``
             :data:`rcParam<matplotlib.rcParams>`.
 
-          ncol : integer
-            number of columns. default is 1
+        ncol : integer
+            Number of columns. Default is 1.
 
-          mode : [ "expand" | *None* ]
-            if mode is "expand", the legend will be horizontally expanded
+        mode : "expand" or None
+            If mode is "expand", the legend will be horizontally expanded
             to fill the axes area (or *bbox_to_anchor*)
 
-          title : string
-            the legend title
+        title : string
+            The legend title
+
+        borderpad : float or None
+            The fractional whitespace inside the legend border.
+            Measured in font-size units.
```

Comment:
This and the 2 comments below are on bits of docstring directly copied from axes.legend(), so should probably fix them there at some point too.